third_party/move/move-analyzer/src/symbols.rs

```diff
@@ -171,12 +170,13 @@ struct StructDef {
     field_defs: Vec<FieldDef>,
 }
 
-#[derive(Derivative, Debug, Clone, PartialEq, Eq)]
-#[derivative(PartialOrd, Ord)]
+#[derive(Derivative)]
+#[derivative(Debug, Clone, Eq, PartialEq, PartialOrd, Ord)]
 pub struct FunctionDef {
     name: Symbol,
     start: Position,
     attrs: Vec<String>,
+    #[derivative(PartialEq = "ignore")]
```

Comment:
The implementations `PartialEq`, `PartialOrd`, and `Ord` should agree with each other: https://docs.rs/rust-libcore/latest/core/cmp/trait.PartialOrd.html#how-can-i-implement-partialord